### DISCUSSION TITLE
ERRO

### DISCUSSION THREAD

**Discussion by ctilumen** [community/community#69814] [Actions]
### Select Topic Area

Bug

### Body

Boa tarde, o github esta assim faz alguns dias, vou subir um arquivo pequeno de menos de 1MB, demora muito tempo para subir por completo, fora que esta dizendo que tenho pouco espaço disponível de armazenamento, sendo que falta 70MB para encher, e ele não sobe mais.
Segue o print.

Ficou uma bosta o github agora, antes era bem melhor.

Alguém pode ajudar?

![Screenshot_1](https://github.com/community/community/assets/25119630/c7c03ae5-97aa-4574-bb6d-3cd79931262d)

**Comment by ctilumen** [comment]
Alguém pode ajudar?

**Reply by jjrh92** [reply]
El limite es mas alto que su mensaje de error. Por lo tanto el problema pudiera ser otro. 

Veo que en varias carpetas hay algunos pdf que pueden ser "pesados". Sin embargo no veo que suponga algun problema ya que la capacidad sigue siendo estandar. Revisa 

git rm -r --cached .
git add .
git commit -m 'git cache cleared'
git push

A ver si esto funciona

**Comment by ctilumen** [comment]
eu tentei mais deu isso no terminal.

![Screenshot_1](https://github.com/community/community/assets/25119630/4d927530-d3dc-4903-a810-0666962039bb)

**Reply by jjrh92** [reply]
Por favor escribe git status y envia screenshot para entender mejor el contexto.

**Comment by ctilumen** [comment]
![Screenshot_2](https://github.com/community/community/assets/25119630/9ce6c017-6896-432e-bb0d-640286916c31)

nem pelo git desktop esta indo mais

**Reply by jjrh92** [reply]
Intenta buscar y eliminar un archivo con extension .lock el cual deberia estar en la carpeta .git del proyecto. Despues que lo elimines trata de hacer commit/push nuevamente.


Pasos.

Ve a la carpeta raiz del proyecto en el explorador de windows.
Ve a opciones de carpeta o "folder options" y marca la opcion que diga "mostrar archivos ocultos" ó "show hidden files".

Ahi encontraras una carpeta .git adentro busca el archivo .lock y prueba haciendo commit/push nuevamente.

**Comment by ctilumen** [comment]
encontrei já, vou tentar fazer esse código para ver se resolve ai volto aqui para comentar, vai demorar um pouquinho, pois o github esta demorando para subir arquivos

**Comment by ctilumen** [comment]
eu não fiz o procedimento citado acima, removi algumas fotos e subiu, só que queria saber porque o github demora tanto tempo assim para subir arquivos?
Vou ter que excluir mais coisas caso eu queira colocar algo mais no site.
![Screenshot_1](https://github.com/community/community/assets/25119630/6af8cc95-ace3-45c1-9ecb-b284da23871c)

**Reply by jjrh92** [reply]
Eso esta bien. Lo que pasa es que github en si no funciona muy bien como hosting, yo lo que le puediera recomendar es que los archivos pdf ù otros archivos que sean muy pesados subirlos a un servidor externo como dropbox ó google drive etc. De esta manera aligera la carga del servidor, tambien el tiempo de subida influye mucho la velocidad de conexion (en teoria). Ya esta todo funcionando ok todo bien cierto?.

**Comment by ctilumen** [comment]
é isso que estou fazendo, estou migrando as fotos tudo para o google photos, e depois vou colocar os pdfs no drive. Obrigado por enquanto

**Reply by jjrh92** [reply]
Me alegra que pudiste solucionar. Saludos desde Chile, por favor no olvides marcar esta conversación como solucionada.

**Reply by jjrh92** [reply]
Aun aparece unanswered, por favor marca la como answered, te debe aparecer en color verde.

![image](https://github.com/community/community/assets/48032098/ad436c2c-ef8b-4565-b4ec-863d89f0f3f1)
